scripts/uninstall.sh

```diff
@@ -33,34 +26,6 @@ TMP_FUNCTION_DIR=$(mktemp -d)
 echo -e "\033[1;34mDynatrace function for Google Cloud Platform monitoring / uninstall script"
 echo -e "\033[0;37m"
 
-if ! command -v yq &> /dev/null
```

Comment:
it is also required to check here if yq and jq exist

Reply:
init_ext_tools does this check